Library/Homebrew/rubocops/cask/url.rb

```diff
@@ -0,0 +1,37 @@
+# typed: true
```

Comment:
Sorbet doesn't run successfully. Need to read up on https://sorbet.org/docs/requires-ancestor.

Reply:
Reading https://blog.appsignal.com/2021/01/13/using-mixins-and-modules-in-your-ruby-on-rails-application.html and https://sorbet.org/docs/requires-ancestor, I do not understand what it wants me to do. As the person who introduced this `requires_ancestor` thing, @dduugg can you shed any light (either here or in Slack)?

Reply:
```ruby
module RuboCop::Cop::Cask::OnUrlStanza
  requires_ancestor { RuboCop::Cop::Cask::UrlLegacyCommaSeparators }
end
```

This code means that any module that includes `RuboCop::Cop::Cask::OnUrlStanza` must subclass `RuboCop::Cop::Cask::UrlLegacyCommaSeparators`. This makes the methods defined in the latter available in the former. (In this specific case, `OnUrlStanza` makes use of `on_url_stanza`, the lone method defined in `UrlLegacyCommaSeparators`.)

Reply:
Hmmm, so if we now have `RuboCop::Cop::Cask::Url` and `RuboCop::Cop::Cask::UrlLegacyCommaSeparators` cops that have the `on_url_stanza` method , which one should inherit from which? They're separate cops.

Reply:
I figured it out!